core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -184,8 +184,12 @@ private function addImageNode(): NodeDefinition
                             ->info('Allows to disable the layer flattening of animated images. Set this option to false to support animations. It has no effect with Gd as Imagine service.')
                         ->end()
                         ->scalarNode('interlace')
+                            ->info('One of the Imagine\Image\ImageInterface::INTERLACE_* constants.')
                             ->defaultValue(ImageInterface::INTERLACE_PLANE)
                         ->end()
+                        ->scalarNode('resampling-filter')
```

Comment:
```suggestion
                        ->scalarNode('resampling_filter')
```

All other keys use underscores, so maybe this one should, too?

Reply:
This comes from Imagine. They use underscores for most options, but for some they use dashes 🙈
See https://github.com/contao/contao/pull/7595#discussion_r1784132093

Reply:
I see …